.gitignore

```diff
@@ -105,6 +105,9 @@
 /dev/Bin64
 /dev/Bin64vc141
 /dev/Bin64vc141.Debug
+/dev/Bin64vc142
+/dev/Bin64vc142.Debug.Dedicated
```

Comment:
replacing all /dev/BinXXXXX entries with a single /dev/Bin* would be simpler and handle for future versions.